### PR TITLE
To go riak ping

### PR DESCRIPTION
**Pull request by rob05c**

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/ping/keys.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	defer inf.Close()
+
+	pingResp, err := riaksvc.Ping(inf.Tx.Tx, inf.Config.RiakAuthOptions)
+	if err != nil {
+		api.HandleErr(w, r, http.StatusInternalServerError, nil, errors.New("error pinging Riak keys: "+err.Error()))
+		return
+	}
+	*inf.CommitTx = true
+	api.WriteResp(w, r, pingResp.Status)
+}
```
Comment (alficles) [inline traffic_ops/traffic_ops_golang/ping/keys.go]: I'm confused. Is this function supposed to be identical to the Riak function? Am I just missing something?

Comment (rob05c) [reply traffic_ops/traffic_ops_golang/ping/keys.go]: Identical, except it directly returns the status string. Yes, it reproduces the existing API. I make no warranty as to the quality or sanity of the existing API.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1797/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1807/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1849/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/1903/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2047/
Test PASSed.

**Pull request merged**